klt-cert-manager/chart/values.yaml

```diff
@@ -9,6 +30,15 @@ tolerations: []
 ## @param     topologySpreadConstraints add topology constraints for cert manager
 topologySpreadConstraints: []
 
+## @param     kubernetesClusterDomain overrides cluster.local
+kubernetesClusterDomain: cluster.local
+## @param     annotations add deployment level annotations
+annotations: {}
+## @param     podAnnotations adds pod level annotations
+podAnnotations: {}
+## @param  schedulingGatesEnabled enables the scheduling gates in lifecycle-operator. This feature is available in alpha version from K8s 1.27 or 1.26 enabling the alpha version
+schedulingGatesEnabled: false
```

Comment:
do we need the `schedulingGatesEnabled` parameter in the cert manager, or can this be removed?

Reply:
you are right this was probably a copypasta! Good catch :)